src/lib/isMobilePhone.js

```diff
@@ -94,6 +94,7 @@ const phones = {
   'fo-FO': /^(\+?298)?\s?\d{2}\s?\d{2}\s?\d{2}$/,
   'fr-BF': /^(\+226|0)[67]\d{7}$/,
   'fr-BJ': /^(\+229)\d{8}$/,
+  'fr-CD': /^(\+?243|0)?(8|9)\d{8}$/,
```

Comment:
If I check the [ITU numbering plan](https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000370001PDFE.pdf) (page 1), it looks like the regexp here would be too permissive.

Kindly check the ranges that were officially announced in that document and adjust your regexp accordingly please.

Reply:
Hey @pano9000 this is a very old document 😂 (from 2009). Some operators in this list are no longer operational. And there are even new operators with different numbering structure. 

But I think I can come up with something like `/^(\+?243|0)?(8|9)(0|1|2|4|7|8\9)\d{7}$/`, so telephone numbers like +243 902 XXXXXXX can pass the regex.

Reply:
The most recent document provided to ITU is from 2013; https://www.itu.int/oth/T0202000037/en
That document is less clear than those of some other countries but looking at the communication of 15.IX.2010 I would propose the RegExp to be the following; `/^(\+?243|0)?(8[0-2|4|8|9]|9[7-9])\d{7}$/`.
`+243 902 XXXXXXX` does not seem to be valid from the official source of the ITU, but if you can provide one from the national authorities that do show that this should be valid we can look at a different RegExp

Reply:
> The most recent document provided to ITU is from 2013; https://www.itu.int/oth/T0202000037/en That document is less clear than those of some other countries but looking at the communication of 15.IX.2010 I would propose the RegExp to be the following; `/^(\+?243|0)?(8[0-2|4|8|9]|9[7-9])\d{7}$/`. `+243 902 XXXXXXX` does not seem to be valid from the official source of the ITU, but if you can provide one from the national authorities that do show that this should be valid we can look at a different RegExp

Hey @WikiRik. My actual phone number is `+243 902 575 693` from [Africel ](https://www.africell.cd/)  😂